packages/nextjs/app/builders/0x8488B81757074B0558ebB05D59A81F43F55bf0C8/page.tsx

```diff
@@ -0,0 +1,50 @@
+"use client";
+
+const GunerivBuilderPage = () => {
+  return (
+    <div className="flex justify-center items-center min-h-screen p-1">
+      <div className="flex justify-center items-center md:items-start flex-col">
+        <div className="flex flex-col justify-center items-center mb-10">
+          <h1 className="text-xl font-bold mb-2 text-white w-full text-center md:text-start">Volkan Guneri</h1>
+          <p className="text-sm mt-1 text-blue-200 w-full">Fullstack Smart Contract Developer</p>
+        </div>
+        <div className="flex flex-col md:flex-row justify-center items-center gap-10">
+          <p className="flex-grow text-blue-100 text-justify leading-relaxed max-w-[35em]">
+            Hi! I&apos;m a full-stack smart contract developer actively seeking new open-source contributions,
+            participating in hackathons, and exploring job opportunities within the captivating Web3 domain. I&apos;m
+            enthusiastic about daily learning and growth in this dynamic sector. Don&apos;t hesitate to get in touch
+            with me for any discussion regarding smart contract development.
+          </p>
+          <ul className="flex flex-row md:flex-col gap-3">
+            <li>
+              <a className="underline pointer" href="https://volkanguneri.github.io/portfolio-front/" target="_blank">
+                Portfolio
```

Comment:
You might consider adding icons here to make it more appealing for the user. Instead of having Portfolio, Github, LinkedIn, etc.